### DISCUSSION TITLE
Converting vector graphics file to gds

### DISCUSSION THREAD

**Discussion by Hohniker** [gdsfactory/gdsfactory#1958] [Q&A]
Is there a simple way of converting a vector graphics file to a gds file using gdsfactory?

**Comment by joamatab** [comment]
not that I know of, we support images as numpy arrays

`gf.read.from_np()`

**Comment by tvt173** [comment]
interesting... people often want to go gds -> svg, but rare to go the other way...

i think this package may be helpful: https://github.com/nvictus/svgpath2mpl

in particular, check out this example:
https://nbviewer.org/github/nvictus/svgpath2mpl/blob/master/examples/tiger.ipynb

**Comment by Hohniker** [comment]
I'll definitely try this. Thanks.

I'm curious why this is so rare, since inserting a company logo into a chip
design is common practice.

**Reply by tvt173** [reply]
makes sense. please share the solution if you get it working!

**Reply by tvt173** [reply]
the other option would be to convert the svg to a png (or other image format), then use Joaquin's solution above for importing an image

**Reply by joamatab** [reply]
i added an example here
https://github.com/gdsfactory/gdsfactory/pull/1976

that will show here
https://gdsfactory.github.io/gdsfactory/notebooks/04_components_geometry.html
